test/models/job_execution_test.rb

```diff
@@ -288,14 +288,14 @@ def last_line_of_output
   it 'can run with a block' do
     x = :not_called
     execution = JobExecution.new('master', job) { x = :called }
-    assert execution.send(:run)
+    execution.send(:run)
```

Comment:
so it fails now ? ... that seems fishy ...

Reply:
It didn't seem like we were using the return value of `status!` anywhere except for these tests, so it seemed unnecessary. I've added it back in.

Reply:
afaik it was in the tests to make sure it did not crash to have a fast indicator for 'something blew up' instead of just missed `expects`